pinot-server/src/main/java/org/apache/pinot/server/starter/helix/HelixInstanceDataManagerConfig.java

```diff
@@ -29,6 +29,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import static org.apache.pinot.common.utils.fetcher.BaseSegmentFetcher.*;
```

Comment:
These imports cause check-style failures flagging static imports.
Should these variables be recreated here ?

Reply:
I think the problem is with the `*`. Maybe you should change your IDE setting and import each explicitly.